iroh-net/Cargo.toml

```diff
@@ -63,7 +63,7 @@ webpki = { package = "rustls-webpki", version = "0.101.4", features = ["std"] }
 webpki-roots = "0.25"
 wg = "0.3.1"
 quinn = "0.10"
-quinn-proto = "0.10"
+quinn-proto = ">=0.10.5"
```

Comment:
What's the >= do here? I thought these versions were minimum verrsions even without the >=...

Reply:
me being overly explicit :) [2f7d427](https://github.com/n0-computer/iroh/pull/1510/commits/2f7d4273c8d6600b67d18f7ea698df13479c31d6)